ingress/controllers/nginx/Makefile

```diff
@@ -4,8 +4,16 @@ all: push
 TAG = 0.5
 PREFIX = gcr.io/google_containers/nginx-ingress-controller
 
+REPO_INFO=$(shell git config --get remote.origin.url)
```

Comment:
Oh, i thought you were building with a specific commit hash of godep. I guess you'er just piping the current commit hash down into the controller.